### PR TITLE
Improved RSpec examples based on suggestions from #895

### PR DESCRIPTION
**Pull request by pkuczynski**

### CODE REVIEW

File: docs/Reek-Driven-Development.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ But there's another way; a much more effective "Reek-driven" approach: add Reek
 
 ```Ruby
 it 'contains no code smells' do
-  Dir['lib/**/*.rb'].should_not reek
+    Pathname.glob('lib/**/*.rb').each do |file|
```
Comment (troessner) [inline docs/Reek-Driven-Development.md]: The indentation is off here, should be two spaces.

File: lib/reek/spec.rb [hunk 2]
```diff
@@ -16,17 +16,19 @@ module Reek
...
   #      end
   #    end
   #  end
   #
   # And here's an even simpler way to do the same:
   #
   #  it 'has no code smells' do
-  #    Dir['lib/**/*.rb'].should_not reek
+  #    Pathname.glob('lib/**/*.rb').each do |path|
+  #        expect(path).not_to reek
```
Comment (troessner) [inline lib/reek/spec.rb]: Indentation is off here as well.

File: docs/Reek-Driven-Development.md (hunk 1 above)
Comment (pkuczynski) [inline docs/Reek-Driven-Development.md]: Indeed! Sorry... Fixed

**Comment by troessner** [comment]
One small comment left and then lgtm!

File: lib/reek/spec.rb (hunk 2 above)
Comment (pkuczynski) [inline lib/reek/spec.rb]: Fixed

**Comment by troessner** [comment]
Merged, thanks a lot!

**Comment by troessner** [comment]
I screwed up. We require atomic commits (== 1 commit per pull request) and I had missed that you hadn't squashed your commits so I just merged 3 commits. Mea culpa @mvz and @chastell

**Comment by mvz** [comment]
No ptoblem @troessner.

**Comment by mvz** [comment]
GitHub should have a 'sent from my mobile device' annotation for comments :-D.

**Comment by chastell** [comment]
I really dislike this practice (it loses information and makes debugging harder for no clear benefit), so don’t worry – and no need to apologise to me. :heart: :laughing:

**Comment by troessner** [comment]
>> I really dislike this practice (it loses information and makes debugging harder for no clear benefit)

@chastell I give at least 4 reasons [here](https://troessner.wordpress.com/2014/05/12/why-you-should-squash-your-commits-and-how-to-do-that/) :)

**Comment by mvz** [comment]
@troessner @chastell I posted a comment there.

**Comment by chastell** [comment]
@troessner The link (along with your blog) seems to have went AWOL. `/o\`

**Comment by troessner** [comment]
>> @troessner The link (along with your blog) seems to have went AWOL.

Yep, I moved to a [new](https://troessner.svbtle.com/), non-shitty blog (wordpress, oh my) yesterday. 
The post above got lost because I plan to rewrite it - I'll then re-link it here.
Sorry about the lost comments there, there was just no way to move them from wordpress.com to svbtle.com ...

**Comment by troessner** [comment]
Link updated above, again, sorry for the lost comments :(

**Comment by mvz** [comment]
Oh noes, you lost my well reasoned and carefully worded comment! :sob: :stuck_out_tongue_closed_eyes:

**Pull request merged**